msteams-platform/concepts/design/design-teams-app-advanced-ui-components.md

```diff
@@ -12,7 +12,7 @@ The following components are a combination of [basic UI components](~/concepts/d
 
 ## Microsoft Teams UI Kit
 
-Based on <a href="https://fluentsite.z22.web.core.windows.net/" target="_blank">Fluent UI</a>, the Microsoft Teams UI Kit includes components and patterns that are designed specifically for building Teams apps. In the UI kit, you can grab and insert the components listed here directly into your design and see more examples of how to use each component.
+Based on [Fluent UI](https://fluentsite.z22.web.core.windows.net/0.64.0), the Microsoft Teams UI Kit includes components and patterns that are designed specifically for building Teams apps. In the UI kit, you can grab and insert the components listed here directly into your design and see more examples of how to use each component.
```

Comment:
Add the URL as https://fluentsite.z22.web.core.windows.net/ and without the version at the end. The URL redirects to v0.64.0 for now. Tomorrow, it may redirect to some other newer version. Why link explicitly to v0.64.0? The version will be hardcoded and will become either a broken link tomorrow OR become a stale link pointing to an older version tomorrow. We don't want both scenarios.